corehq/apps/fixtures/resources/v0_1.py

```diff
@@ -62,3 +63,16 @@ class Meta(CustomResourceMeta):
         object_class = FixtureDataItem
         resource_name = 'fixture'
         limit = 0
+
+
+class InternalFixtureResource(FixtureResource):
```

Comment:
Did you verify that this is actually used internally? The one I was thinking of was the cloudcare API that gets the fixture XML for a user. I'm not sure if this one is